src/main/java/com/openshift3/client/authorization/BearerTokenAuthorizationStrategy.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package com.openshift3.client.authorization;
+
+/**
+ * Authorization stategy to add a Bearer Token to a request
+ */
+public class BearerTokenAuthorizationStrategy implements IAuthorizationStrategy {
+
+	public static final String AUTHORIZATION = "Authorization";
+
+	private final String token;
+	
+	public BearerTokenAuthorizationStrategy(String token) {
+		this.token = token;
+	}
+
+	@Override
+	public void authorize(IAuthorizationRequest request) {
```

Comment:
Is that not simply a general purpose request (rather than an authorization request?). What's the advantage of this more specific type?